Orange/widgets/data/owfile.py

```diff
@@ -74,13 +81,12 @@ class OWFile(widget.OWWidget, RecentPathsWComboMixin):
         doc="Attribute-valued data set read from the input file.")]
 
     want_main_area = False
-    resizing_enabled = False
```

Comment:
Now that resizing is enabled again:
does line L174 `url_combo.setMaximumWidth(500)` still make sense?

`file_combo` does not have a max set and extends to fill the widget, which would be even more useful for potentially long URLs...